openedx/features/student_certificates/handlers.py

```diff
@@ -1,40 +1,56 @@
+"""
+Handlers for different signals in the student_certificates application.
+"""
 import logging
 
 from django.db.models.signals import post_save
 from django.dispatch import receiver
 
 from common.lib.mandrill_client.client import MandrillClient
 from lms.djangoapps.certificates.models import GeneratedCertificate
-
-from models import CertificateVerificationKey
+from openedx.features.student_certificates.models import CertificateVerificationKey
 from openedx.features.student_certificates.signals import USER_CERTIFICATE_DOWNLOADABLE
-from tasks import task_create_certificate_img_and_upload_to_s3
+from openedx.features.student_certificates.tasks import task_create_certificate_img_and_upload_to_s3
 
 log = logging.getLogger(__name__)
 
 
 @receiver(post_save, sender=GeneratedCertificate)
-def generate_certificate_img(instance, created, **kwargs):
+def generate_certificate_img(instance, created, **_kwargs):
     if not created:
         task_create_certificate_img_and_upload_to_s3.delay(instance.verify_uuid)
 
 
 @receiver(post_save, sender=GeneratedCertificate)
-def generate_certificate_verification_key(instance, created, **kwargs):
+def generate_certificate_verification_key(instance, created, **_kwargs):
     if created and not hasattr(instance, 'certificate_verification_key'):
         CertificateVerificationKey.objects.create_object(instance)
 
 
 @receiver(USER_CERTIFICATE_DOWNLOADABLE)
 def send_email_user_certificate_downloadable(sender, first_name, display_name, certificate_reverse_url, user_email,
-                                             *args, **kwargs):
+                                             *_args, **_kwargs):  # pylint: disable=unused-argument
```

Comment:
Let's please follow the edx [recommendations](https://edx.readthedocs.io/projects/edx-developer-guide/en/latest/style_guides/python-guidelines.html#breaking-long-lines) regarding breaking long lines.